src/dropdown.test.interactions.single.ts

```diff
@@ -445,6 +610,30 @@ it('selects and deselects options when `value` is set programmatically', async (
   expect(options[1]?.selected).to.be.true;
 });
 
+it('only selects the first option when `value` is set programmatically and multiple options have the same `value`', async () => {
```

Comment:
> When `value` is set programmatically, only the first Dropdown Option with a matching value is now selected instead of every Dropdown Option with a matching value.